services/device-registry-mongodb/src/main/java/org/eclipse/hono/deviceregistry/mongodb/ApplicationConfig.java

```diff
@@ -255,6 +275,17 @@ public MongoDbBasedCredentialsService credentialsService() {
         return new MongoDbBasedCredentialsService();
     }
 
+    /**
+     * Exposes the MongoDB tenant service as a Spring bean.
+     *
+     * @return The MongoDB registration service.
```

Comment:
Tenant service and not registration service. Could you please update it?

Reply:
done